docs/getting-started/add-sample-data.asciidoc

```diff
@@ -1,22 +1,23 @@
 [[add-sample-data]]
 == Get up and running with sample data
 
-{kib} has three sample data sets that you can use to explore {kib} before loading your own data 
-source. Each set is prepackaged with a dashboard of visualizations and a 
-{kibana-ref}/canvas-getting-started.html[Canvas workpad].
+{kib} has several sample data sets that you can use to explore {kib} before loading your own data.
+Sample data sets install prepackaged visualizations, dashboards,
+{kibana-ref}/canvas-getting-started.html[Canvas workpads],
+and {kibana-ref}/maps.html[Maps].
 
-The sample data sets address common use cases:
```

Comment:
I changed the working from `common use cases` to `variety of use cases` because the word `common` to me installs a feeling of limit. I like `variety` better because it implies there are so many more.